client/checkout/upe-styles/index.js

```diff
@@ -489,24 +568,32 @@ export const getAppearance = ( elementsLocation, forWooPay = false ) => {
 		fontSizeBase: labelRules.fontSize,
 	};
 
-	const appearance = {
+	let appearance = {
 		variables: globalRules,
 		theme: isColorLight( backgroundColor ) ? 'stripe' : 'night',
-		rules: {
-			'.Input': inputRules,
-			'.Input--invalid': inputInvalidRules,
-			'.Label': labelRules,
-			'.Block': blockRules,
-			'.Tab': tabRules,
-			'.Tab:hover': tabHoverRules,
-			'.Tab--selected': selectedTabRules,
-			'.TabIcon:hover': tabIconHoverRules,
-			'.TabIcon--selected': selectedTabIconRules,
-			'.Text': labelRules,
-			'.Text--redirect': labelRules,
-		},
+		labels: isFloatingLabel ? 'floating' : 'above',
```

Comment:
```suggestion
		labels: 'blocks_checkout' === elementsLocation  ? 'floating' : 'above',
```
I'm not ostensibly opposed to adding a new parameter to `getAppearance`, but is there any other scenario other than the blocks checkout where this would be used? Or alternatively are there scenarios where `elementsLocation` might be `blocks_checkout` and we would _not_ want to display floating labels? 🤔